2022-03-08 21:34:18,738 INFO    MainThread:14652 [wandb_setup.py:_flush():71] setting env: {}
2022-03-08 21:34:18,739 INFO    MainThread:14652 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-08 21:34:18,739 INFO    MainThread:14652 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\PC\Desktop\Wave-Defense-Learning-Environment\wandb\run-20220308_213418-282up8qq\logs\debug.log
2022-03-08 21:34:18,739 INFO    MainThread:14652 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\PC\Desktop\Wave-Defense-Learning-Environment\wandb\run-20220308_213418-282up8qq\logs\debug-internal.log
2022-03-08 21:34:18,739 INFO    MainThread:14652 [wandb_init.py:init():404] calling init triggers
2022-03-08 21:34:18,739 INFO    MainThread:14652 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'gamma': 0.99, 'log_interval': 25, 'episodes': 1000000, 'num_steps': 1000, 'batch_size': 128, 'lr': 0.001, 'eps_start': 1.0, 'eps_end': 0.1, 'eps_decay': 50000, 'target_update': 5000, 'replay_size': 2000}
2022-03-08 21:34:18,739 INFO    MainThread:14652 [wandb_init.py:init():460] starting backend
2022-03-08 21:34:18,739 INFO    MainThread:14652 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2022-03-08 21:34:18,743 INFO    MainThread:14652 [backend.py:ensure_launched():216] starting backend process...
2022-03-08 21:34:18,790 INFO    MainThread:14652 [backend.py:ensure_launched():222] started backend process with pid: 7064
2022-03-08 21:34:18,791 INFO    MainThread:14652 [wandb_init.py:init():469] backend started and connected
2022-03-08 21:34:18,791 INFO    MainThread:14652 [wandb_init.py:init():533] updated telemetry
2022-03-08 21:34:18,825 INFO    MainThread:14652 [wandb_init.py:init():563] communicating current version
2022-03-08 21:34:19,731 INFO    MainThread:14652 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-08 21:34:19,732 INFO    MainThread:14652 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-08 21:34:19,960 INFO    MainThread:14652 [wandb_init.py:init():606] starting run threads in backend
2022-03-08 21:34:22,578 WARNING MainThread:14652 [wandb_init.py:init():952] interrupted
Traceback (most recent call last):
  File "C:\Users\PC\anaconda3\envs\ballsort3.7\lib\site-packages\wandb\sdk\wandb_init.py", line 931, in init
    run = wi.init()
  File "C:\Users\PC\anaconda3\envs\ballsort3.7\lib\site-packages\wandb\sdk\wandb_init.py", line 611, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "C:\Users\PC\anaconda3\envs\ballsort3.7\lib\site-packages\wandb\sdk\interface\interface.py", line 221, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "C:\Users\PC\anaconda3\envs\ballsort3.7\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 423, in _communicate_run_start
    result = self._communicate(rec)
  File "C:\Users\PC\anaconda3\envs\ballsort3.7\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 213, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "C:\Users\PC\anaconda3\envs\ballsort3.7\lib\site-packages\wandb\sdk\interface\router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "C:\Users\PC\anaconda3\envs\ballsort3.7\lib\threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\PC\anaconda3\envs\ballsort3.7\lib\threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
